.github/workflows/ci.yml

```diff
@@ -17,6 +17,7 @@ jobs:
           - gemfiles/rails7_1.gemfile
           - gemfiles/rails7_2.gemfile
           - gemfiles/rails8_0.gemfile
+          - gemfiles/deps.gemfile
```

Comment:
would rails8_0_sidekiq_7_0 be more descriptive filename?

Reply:
Yes. Maybe.

I was thinking leaving this as a single file where specific dependencies can be pinned to lower versions. The rails version doesn't really matter, I just wanted to set it.

If we have gemfiles for every dep that gets pinned it gets multiplied by every ruby version and I we end up with lots of runs. I don't know if there are limits. 

I can name it something more descriptive though.